sources/packages/api/src/database/entities/application.model.ts

```diff
@@ -140,4 +140,10 @@ export class Application extends RecordDataModel {
     type: "enum",
   })
   coeStatus: COEStatus;
+
+  @Column({
+    name: "application_status_updated_on",
+    nullable: true,
```

Comment:
Since the column is not null on DB, please remove it.

Reply:
Ya, nice catch